x509/x509.go

```diff
@@ -200,6 +210,7 @@ const (
 	SHA256WithRSAPSS
 	SHA384WithRSAPSS
 	SHA512WithRSAPSS
+	ED25519SIG
```

Comment:
That's not the right const name is it? Though, a quick search isn't turning up any -- so maybe there isn't a "standard" name here, in which case `ED25519Sig` would match the camel case of the rest (the `Sig` suffix itself is inconsistent, but I don't know how else you would distinguish it from just `ED25519`)